deploy/crds/shipwright.io_builds.yaml

```diff
@@ -21,7 +21,8 @@ spec:
       jsonPath: .status.registered
       name: Registered
       type: string
-    - description: The reason of the registered Build, either an error or succeed message
+    - description: The reason of the registered Build, either an error or succeed
```

Comment:
This text wrapping seems to be cropping up again, after being removed in #786.

Can you give me details of how you're running `make generate-crds` to produce this? What version of `controller-gen` are you using? Is it being done automatically by an IDE?

Reply:
I went ahead and reran make generate-crds but the same text wrapping item occurred

Reply:
@imjasonh bump on this ... any clues on controller-gen "text wrapping" .... aside from mining help (with no luck), and finding the upstream issue on this, which when reading did see you previously noted in #786 I'm not seeing a clear resolution path yet

perhaps an annoyance we'll just have to live with for now ...

Reply:
worked around this ... see https://github.com/shipwright-io/build/pull/809#issuecomment-870936103